moveit_ros/planning/trajectory_execution_manager/src/trajectory_execution_manager.cpp

```diff
@@ -1750,7 +1764,7 @@ bool TrajectoryExecutionManager::ensureActiveControllers(const std::vector<std::
 void TrajectoryExecutionManager::loadControllerParams()
 {
   XmlRpc::XmlRpcValue controller_list;
```

Comment:
I didn't know XmlRpcValue was a valid parameter type?

Reply:
Commented this out for a later code base refactor.